### ISSUE TITLE
Missing syntax error for VALUE operator

### ISSUE THREAD

**Issue by smardcode** [abaplint/abaplint#1746]
```abap
TYPES ty_strings TYPE STANDARD TABLE OF string WITH EMPTY KEY.
DATA(words) = VALUE ty_strings(
  ( `All` )
  ( `ABAP` )
  ( `constructs` )
  ( `are` )
  ( `imperative` ) ).

"Table comprehension into helper table
DATA(switched_words) = VALUE i_do_not_exist(
      FOR word IN words
        ( SWITCH #( word WHEN `All`        THEN `Some`
                        WHEN `imperative` THEN `functional`
                        ELSE word ) ) ).

WRITE switched_words.
```
The linter does not complain about the missing type "i_do_not_exist":
![image](https://user-images.githubusercontent.com/60311033/107127986-66a18800-68ba-11eb-8103-7d8812811de8.png)

![image](https://user-images.githubusercontent.com/60311033/107127992-7325e080-68ba-11eb-9c9a-1c70cc72a0e0.png)

**Comment by larshp** [comment]
the name `i_do_not_exist` is outside the `errorNamespace`, ie. it becomes void

if you want errors for everything, the linter must know everything, then change the errorNamespace to blank = regex matches everything

![image](https://user-images.githubusercontent.com/5888506/107199998-6eb31200-69f7-11eb-9166-8264bdfb3319.png)

**Comment by larshp** [comment]
closing
